golem-worker-service/src/api/definition/patterns.rs

```diff
@@ -0,0 +1,48 @@
+#[derive(Debug)]
+pub struct VarInfo {
+    pub key_name: String,
+}
+
+#[derive(Debug)]
+pub struct CatchAllVarInfo {
+    pub key_name: String,
+}
+
+#[derive(Debug)]
+pub enum PathPattern {
+    Literal(String),
+    Var(VarInfo),
+    CatchAllVar(CatchAllVarInfo),
+}
+
+#[derive(Debug)]
+pub struct AllPathPatterns {
+    pub path_patterns: Vec<PathPattern>,
+}
+
+impl AllPathPatterns {
```

Comment:
Is this simply reorganization of the existing code? Or are you introducing new parsing here?

Reply:
Its reorganization of the existing code